### ISSUE TITLE
Streaming Read?

### ISSUE THREAD

**Issue by pwalsh** [guyonroche/exceljs#33]
Hi, from the documentation, it is not clear (to me). Does this lib support streaming reads of sheets in an Excel workbook? I want to work with potentially huge sheets of data without loading all into memory.

**Comment by MrHubble** [comment]
@pwalsh can you please share some sample code for how you implemented this?

**Comment by pwalsh** [comment]
@MrHubble I didn't implement. I'm looking for a Node lib that does, hence my question.

**Comment by alangunning** [comment]
@pwalsh @MrHubble  Did you figure out how to stream read sheets?

**Comment by pwalsh** [comment]
@alangunning no, did not look at it yet.

**Comment by MrHubble** [comment]
@alangunning no, I didn't implement streaming with exceljs either. For now I have restricted the amount of rows my user can import to 1,000 and continue to use https://github.com/SheetJS/js-xlsx

I would prefer to stream with exceljs in the future. Please post your solution if you are successful.

**Comment by guyonroche** [comment]
I'm still working on streaming reads. I ran into a problem with memory for very large sheets. Most likely will have to solve this with the aid of a /tmp folder. I will post here when ready

**Comment by alangunning** [comment]
Thank you. Your response and efforts are much appreciated.
Alan

**Comment by pwalsh** [comment]
@guyonroche very happy to review anything you have, when you are ready.

@rgrp you might want to follow this.
